src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -163,7 +163,8 @@ const useCoreChatSystemHook = () => {
     const channelType: {[key: string]: ChannelType} = {
       messaging: isAnonymous ? 'ANON_PM' : 'PM',
       group: 'GROUP',
-      topics: isAnonymous ? 'ANON_TOPIC' : 'TOPIC'
+      topics: isAnonymous ? 'ANON_TOPIC' : 'TOPIC',
+      topicinvitation: 'TOPIC'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new chat type 'topicinvitation' is mapped to the same ChannelType as 'topics', i.e., 'TOPIC'. If this is intentional and both 'topics' and 'topicinvitation' are meant to be treated the same way, then it's fine. However, if 'topicinvitation' should have a different behavior or properties, you might need to define a new ChannelType for it.
```diff
     const channelType: {[key: string]: ChannelType} = {
       messaging: isAnonymous ? 'ANON_PM' : 'PM',
       group: 'GROUP',
       topics: isAnonymous ? 'ANON_TOPIC' : 'TOPIC',
-      topicinvitation: 'TOPIC'
+      topicinvitation: 'TOPIC_INVITATION' // if a new ChannelType is needed
     };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->